hail/python/test/hail/methods/test_impex.py

```diff
@@ -1223,6 +1223,15 @@ def test_import_locus_intervals_no_reference_specified(self):
         self.assertTrue(t.count() == 2)
         self.assertEqual(t.interval.dtype.point_type, hl.tstruct(contig=hl.tstr, position=hl.tint32))
 
+    def test_import_locus_intervals_recoding(self):
+        interval_file = resource('annotinterall.grch38.no.chr.interval_list')
+        t = hl.import_locus_intervals(interval_file,
+                                      contig_recoding={str(i): f'chr{i}' for i in [*range(22), 'X', 'Y', 'MT']},
```

Comment:
I'd take out MT here. I'm not sure chrMT is a thing in GRCh38.

Reply:
Changed to `M`